src/username_bruteforcer/username_bruteforcer.py

```diff
@@ -49,50 +49,66 @@ def load_params(self):
             self.response_key = key['response']['response_key']
             self.valid_response = key['response']['valid_response']
             self.url = key['url']
-            self.method = key['method'].lower()
             self.data[self.user_field] = ""
 
             try:
                 self.proxies = key['proxies']
             except KeyError:
                 self.proxies = None
 
+            try:
+                self.methods = [m.lower().strip() for m in key['methods']]
+            except KeyError:
+                raise ValueError('Missing "methods" list')
+
             if key['data'] and key['data'] != None:
                 data_vals = key['data']
                 for k in data_vals:
                     self.data[k] = data_vals[k]
 
-            if key['users']:
-                users_list = key['users']
-                self.check_for_valid_users(users_list)
+            try:
+                for m in self.methods:
+                    self.check_for_valid_users(key['users'], m)
+            except KeyError:
+                raise ValueError('Missing "users" list')
 
-    def check_for_valid_users(self, users_list):
+    def check_for_valid_users(self, users_list, method):
         """
         Using input parameters
         test target url/api
         to see if user exists
         """
 
+        self.valid_users[method] = {'users': []}
+        self.invalid_users[method] = {'users': []}
+
         for u in users_list:
 
             self.data[self.user_field] = u
-            requestMethod = getattr(requests, self.method, None)
+            requestMethod = getattr(requests, method, None)
 
             if requestMethod is None:
                 raise ValueError('Method "{method}" is not supported'.format(
-                    method=repr(self.method)))
+                    method=repr(method)))
+
+            if method == "get":
+                r = requestMethod(self.url, params=self.data,
+                                  proxies=self.proxies)
+            else:
```

Comment:
Can we not define the function call once, but pass in the key/value pair dynamically?
i.e. `params=self.data` or `data=self.data`